tests/testthat/helper-html.R

```diff
@@ -3,7 +3,7 @@ show_raw_html <- function(x) {
 }
 
 expect_snapshot_html <- function(x, .envir = parent.frame()) {
-  x_str <- deparse(substitute(x))
+  x_str <- deparse1(substitute(x))
```

Comment:
This might solve some of the problem, but only if you create the toolbar inside `expect_snapshot_html()` because the seed needs to be set when you call `toolbar_input_button()`

```suggestion
  with_private_seed()
  x_str <- deparse1(substitute(x))
```

but this could cover a lot of the cases by default